modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_46890.testjava

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.portal.kernel.service.ServiceContext;
+
+import java.util.Locale;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Regisson Aguiar
+*/
+public class LPD_46890 {
+
+	public void method(
+		long userId, long groupId, long classNameId, long classPK,
+		long resourceClassNameId, Map<Locale, String> nameMap,
+		Map<Locale, String> descriptionMap, String type, String mode,
+		String language, String script, ServiceContext serviceContext) {
+
+		DDMTemplateLocalServiceUtil.addTemplate(123, groupId, classNameId, classPK, resourceClassNameId, nameMap, descriptionMap, type, mode, language, script, serviceContext);
+		DDMTemplateLocalServiceUtil.addTemplate(userId, groupId, classNameId, classPK, resourceClassNameId, nameMap, descriptionMap, type, mode, language, script, serviceContext);
+		_ddmTemplateLocalService.addTemplate(123, groupId, classNameId, classPK, resourceClassNameId, nameMap, descriptionMap, type, mode, language, script, serviceContext);
```

Comment:
@regiCesar21 only one testcase for the suggestion is necessary, can you remove this one please.